Controllers/UserController.cs

```diff
@@ -48,25 +43,19 @@ public async Task<IActionResult> Login(LoginDTO model)
             if (user == null)
                 return ApiBadRequest("User does not exist.");
 
-            var result =
-                await _signInManager.PasswordSignInAsync(user.UserName, model.Password, false, lockoutOnFailure: false);
-            if (result.IsLockedOut)
-                return ApiBadRequest("User account locked out.");
-
-            if (!result.Succeeded)
-                return ApiBadRequest("Invalid username or password.");
-
-            var roles = await _userManager.GetRolesAsync(user);
-
-            return Ok(new
-            {
-                token = _jwt.GenerateSecurityToken(new JwtUser()
+            if (user != null && await _userManager.CheckPasswordAsync(user, model.Password)) {
+                return Ok(new
                 {
-                    Username = user.UserName,
-                    Email = user.Email,
-                    roleId = (DepartmentId)Enum.Parse(typeof(DepartmentId), roles.First(), true)
-                })
-            });
+                    token = _jwt.GenerateSecurityToken(new JwtUser()
+                    {
+                        Username = user.Username,
```

Comment:
If we are not using username for authentification, maybe we shouldn't put it in the jwt as well?

Reply:
But it doesn't hurt that it's there and we are using it.

Reply:
It doesn't provide any relevant information either because once we implement user registration from Admin we will not be using usernames.